templates/expo-template-default/components/themed-text.tsx

```diff
@@ -34,27 +40,44 @@ export function ThemedText({
 }
 
 const styles = StyleSheet.create({
-  default: {
-    fontSize: 16,
-    lineHeight: 24,
+  small: {
+    fontSize: 14,
+    lineHeight: 20,
+    fontWeight: 500,
   },
-  defaultSemiBold: {
+  smallBold: {
+    fontSize: 14,
+    lineHeight: 20,
+    fontWeight: 700,
+  },
+  default: {
     fontSize: 16,
     lineHeight: 24,
-    fontWeight: '600',
+    fontWeight: 500,
   },
   title: {
-    fontSize: 32,
-    fontWeight: 'bold',
-    lineHeight: 32,
+    fontSize: 48,
+    fontWeight: 600,
+    lineHeight: 52,
   },
   subtitle: {
-    fontSize: 20,
-    fontWeight: 'bold',
+    fontSize: 32,
+    lineHeight: 44,
+    fontWeight: 600,
   },
   link: {
     lineHeight: 30,
-    fontSize: 16,
-    color: '#0a7ea4',
+    fontSize: 14,
+    // color: '#3c87f7',
+  },
+  linkPrimary: {
+    lineHeight: 30,
+    fontSize: 14,
+    color: '#3c87f7',
+  },
+  code: {
+    fontFamily: Fonts.mono,
+    fontWeight: Platform.select({ ios: 500, android: 700, web: 500 }),
```

Comment:
Maybe something like this?
```suggestion
    fontWeight: Platform.select({ android: 700 }) ?? 500,
```